### DISCUSSION TITLE
How to display table(s) on home page to non-logged-in (anonymous) users

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#6024] [Help] [answered]
I'm using Filament to prototype a marketplace application for training courses and I'd like the main home page to list some courses (and training providers).

So far I've built all the resources and they're available in the dashboard under /admin.

I've also installed the breezy plugin to handle user registration, etc.

At the moment the home page is the standard Laravel welcome page. I'd like to replace this (or amend it) to:
1. Show a list of courses.
2. Show login and registration links (or the name of the logged in user with a link to their profile).

I've slightly modified the standard Laravel welcome page to show the logged in user name as follows:
```
@if (Route::has('login'))
    <div class="sm:fixed sm:top-0 sm:right-0 p-6 text-right">
        @auth
            <a href="{{ url('/admin/my-profile') }}" class="font-semibold text-gray-600 hover:text-gray-900 dark:text-gray-400 dark:hover:text-white focus:outline focus:outline-2 focus:rounded-sm focus:outline-red-500">{{  Auth::user()->name  }}</a>
        @else
            <a href="{{ route('login') }}" class="font-semibold text-gray-600 hover:text-gray-900 dark:text-gray-400 dark:hover:text-white focus:outline focus:outline-2 focus:rounded-sm focus:outline-red-500">Log in</a>

            @if (Route::has('register'))
                <a href="{{ route('register') }}" class="ml-4 font-semibold text-gray-600 hover:text-gray-900 dark:text-gray-400 dark:hover:text-white focus:outline focus:outline-2 focus:rounded-sm focus:outline-red-500">Register</a>
            @endif
        @endauth
    </div>
@endif
```

I can get the login and registration links to show up by adding the following routes:
```
Route::get('/login', \JeffGreco13\FilamentBreezy\Http\Livewire\Auth\Login::class)->name('login');
Route::get('/register', \JeffGreco13\FilamentBreezy\Http\Livewire\Auth\Register::class)->name('register');
```
The only other route in web.php is for the home page:
```
Route::get('/', function () {
    return view('welcome');
});
```

This gives me a home page for anonymous users like this:
![Screenshot 2023-03-24 at 13 20 45](https://user-images.githubusercontent.com/107488019/227532333-331b9c06-66bf-4af2-8c6e-d011ef18979c.png)

When I log in I'm taken to the Filament dashboard at /admin. Ideally I'd like to stay on the home page. When I manually navigate to the home page it displays the logged in user's name correctly.

So...
1. How do I redirect to the home page after login?
2. How do I show a Filament table on the home page?

**Comment by danharrin** [comment, marked as answer]
To embed a table on a non-admin page, create a Livewire component and [follow these steps](https://filamentphp.com/docs/2.x/tables/getting-started).

**Comment by LemonPie22** [comment]
Okay that makes sense @danharrin, but isn't there an easier solution than creating my own Livewire components? 

Maybe I'm missing something but could I not have Filament on my home page rather than under /admin? And then display tables based on whether the user is logged in or not? Then I could use standard Filament resources rather than delving into Livewire. Or am I completely off track with that idea?

**Reply by danharrin** [reply]
Filament admin panel is for authenticated users only. Our table builder is made to work anywhere. Creating a Livewire component is not difficult, I suggest you give it a go :)

**Reply by LemonPie22** [reply]
Ok will do. Thanks Dan :)

**Reply by LemonPie22** [reply]
One more thing if you don't mind @danharrin: How do I redirect to the home page after login?

**Reply by danharrin** [reply]
Create a new class, extend `Filament\Http\Responses\Auth\LoginResponse`, set your redirect URL, and then bind it to the container with the `Filament\Http\Responses\Auth\Contracts\LoginResponse` interface.
